README.md

```diff
@@ -71,17 +85,29 @@ that are described in this repository have been used in several research
 projects, but there are no guarantees regarding the accuracy or appropriate 
 uses.
 
-Care must be taken to understand how specific variable transformation and harmonization with `cchsflow` affect your study or use of CCHS data. Across 
+Care must be taken to understand how specific variable transformation and 
+harmonization with `cchsflow` affect your study or use of CCHS data. Across 
 survey cycles, almost all CCHS variables have had at least some change in 
-wording and category responses. Furthermore, there have been changes in survey sampling, response rates, weighting methods and other survey design changes that affect responses. 
+wording and category responses. Furthermore, there have been changes in survey 
+sampling, response rates, weighting methods and other survey design changes that 
+affect responses. 
 
 ## Installation
 
-    # If not installed, install the devtools
-    install.packages("devtools")
-    
-    # then, install the package
+```
+    # Install release version from CRAN
```

Comment:
I pushed a similar edit on dev before reviewing PR, hence the merge conflict with README.md. We can keep this version when resolving the merge conflict @DarkShadeKnigh @DougManuel